File: iniciante/python/1064-positivos-e-media.py
def numeros_positivos():
    positivos = soma = i = 0
    while i < 6:
        numero = float(input())
        if numero > 0:
            positivos += 1
            soma += numero
        i += 1
    media = soma / positivos
    print(f'{positivos} valores positivos')
    print(f'{media:.1f}')


numeros_positivos()

File: iniciante/c/1064-positivos-e-media.c
#include <stdio.h>

int main(){
    int i, positivos = 0;
    float numero, media, total = 0.0;
    for (i = 0; i < 6; i++){
        scanf("%f", &numero);
        if (numero > 0.0){
            total += numero;
            positivos++;
        }
    }
    media = total / positivos;
    printf("%d valores positivos\n", positivos);
    printf("%.1f\n", media);

    return 0;
}
